website/docs/r/container_app.html.markdown

```diff
@@ -190,6 +190,8 @@ A `volume` block supports the following:
 
 * `storage_type` - (Optional) The type of storage volume. Possible values are `AzureFile`, `EmptyDir` and `Secret`. Defaults to `EmptyDir`.
 
+* `mount_options` - Mount options used while mounting the AzureFile. Must be a comma-separated string.
```

Comment:
It would be helpful to provide an example of a valid value here e.g. 
```suggestion
* `mount_options` - Mount options used while mounting the AzureFile. Must be a comma-separated string e.g. `dir_mode=0751,file_mode=0751`.
```